### ISSUE TITLE
Resize filter pane

### ISSUE THREAD

**Issue by birbilis** [seajax/seajax#7]
I see the filters pane isn't resizable in width. Can I resize it in the CSS or something at least?

**Comment by sethobrien** [comment]
It's possible to change the width of the side panes to a new constant value by modifying the width of `div.pivot_pane` (PivotView.css line 94). There's not currently any support for dynamic resizing to base the pane sizes on the viewer's total size (which would be nice).

**Comment by birbilis** [comment]
Could it have a CSS min-width there set to 200px
http://www.w3schools.com/cssref/pr_dim_min-width.asp
and have width use "em" or some other unit that is related to how small/big text shows instead?

**Comment by sethobrien** [comment]
Yes, that sounds like a very sensible approach.It looks like a bunch of other sizes would also have to be changed to be relative to the font-size in the containing page, so that text content in the panes also increases or decreases accordingly. Alternately, we could size the pane to the larger of 200px or some percentage of the viewer width on initialization and on every subsequent resize. That percentage would have to live in JS (not CSS), since JS code called by PivotViewer.onResize needs to know the pane width.

**Comment by birbilis** [comment]
Using a percentage for the filter pane size might not show well at big view widths, unless the text also grows dynamically. Most probably "em" would do the job if used consistently in the UI. Not sure what "em" does when view is zoomed (via the browser's zoom feature) though. I think Mozilla and IE also had separate Text Magnification feature (apart from the Zoom one), but not sure if they use it anymore

**Comment by birbilis** [comment]
btw, what I'm currently using at http://gallery.trafilm.net to make the left pane wider is width:292px at .pivot_pane and recently noticed that I also had to change width:255px at .pivot_searchbox (else the search box didn't allow me to type all the way to the available pane width and while typing it was showing a black "x" button [that clears typed text] up to the point where it allowed me to type in, but not up to the available width (since I had made the pivot pane wider)

so it would be nice if the pivot_searchbox didn't have its width hardcoded (other children of the pivot_pane seem to not use hardcoded widths and work fine)
